scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -209,11 +209,15 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
 
   /* ------------- PARSER-SPECIFIC TOKENS -------------------------------------------- */
 
-  sealed trait SepRegion {
+  sealed trait SepRegion { self =>
     def indent = -1
     def closeOnNonCase = false
     def indentOnArrow = true
     def isIndented = false
+    def isEnum: Boolean = self match {
```

Comment:
```suggestion
    def isEnum: Boolean = this match {
```
should be enough, no?